glenn2vcf/build/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:15.10
```

Comment:
Probably not, though I did test it and it works properly. I'll change the runtime environment to 15.10.

The main reason for it is an issue with installation of the correct version of gcc (gcc-4.9) that is only accessible to Ubuntu version 15.10 and later for some reason. It's only needed for compilation and the binary runs and works on 14.04, so the deployment shouldn't be affected by that.

For consistency, it makes sense to have them both be the same version. I did the same thing for the vg tool and will also change it.